### PR TITLE
Fix content transfer encoding

### LINKED ISSUE #50: emailFromId converts = in Query String To ANSI characters
When I try to read an email that contains an URL with a query string the first couple characters after equal sign become decoded by quoted_printable_decode(), for example: 
Email boy like: 

> Check the list on page 56 at http://example.com/list.php?page=56

becomes: 

> Check the list on page 56 at http://example.com/list.php?pageV

Expected: 
URLs containing equal character followed by two hexadecimal characters does not translate into ANSI equivalent

### PR DESCRIPTION
**Pull request by margori**
Fixes #50 

Bug reproduced. Problems in Content-transfer-encoding.

Fixed and added more test cases.

### CODE REVIEW

File: tests/acceptance/MailcatcherCest.php [hunk 1]
```diff
@@ -140,4 +141,28 @@ public function test_grab_urls_from_last_email(
 
         $I->assertEquals($example[0], $urls[0]);
     }
+
+    /**
+     *
+     * @param AcceptanceTester $I
+     * @param \Codeception\Scenario $scenario
+     * @param \Codeception\Example $example
```
Comment (Jamesking56) [inline tests/acceptance/MailcatcherCest.php:149]: Useless `@param`

File: tests/acceptance/MailcatcherCest.php [hunk 2]
```diff
@@ -140,4 +141,28 @@ public function test_grab_urls_from_last_email(
 
         $I->assertEquals($example[0], $urls[0]);
     }
+
+    /**
+     *
```
Comment (Jamesking56) [inline tests/acceptance/MailcatcherCest.php:146]: Random gap?

File: tests/_support/Helper/Acceptance.php [hunk 3]
```diff
@@ -18,4 +18,19 @@ public function sendEmail($to, $subject, $body)
         $phpmailer->Body = $body;
         $phpmailer->send();
     }
+
+    public function sendEmailWithEncoding($to, $subject, $body, $encoding)
```
Comment (Jamesking56) [inline tests/_support/Helper/Acceptance.php:22]: Lots of duplication here, is it worth just updating the above method `sendEmail` and setting a default for the encoding?

**Comment by margori** [comment]
Good comments. Code updated.

File: tests/_support/Helper/Acceptance.php [hunk 4]
```diff
@@ -6,31 +6,19 @@
...
 class Acceptance extends \Codeception\Module
 {
-    public function sendEmail($to, $subject, $body)
+    public function sendEmail($to, $subject, $body, $encoding = '')
     {
         $phpmailer = new \PHPMailer();
         $phpmailer->isSMTP();
         $phpmailer->Host = '127.0.0.1';
         $phpmailer->Port = 1025;
+        if ($encoding) {
```
Comment (Jamesking56) [inline tests/_support/Helper/Acceptance.php:15]: This is a weak check, lets be more strict:

```suggestion
        if (null !== $encoding) {
```

File: tests/_support/Helper/Acceptance.php [hunk 5]
```diff
@@ -6,31 +6,19 @@
 
 class Acceptance extends \Codeception\Module
 {
-    public function sendEmail($to, $subject, $body)
+    public function sendEmail($to, $subject, $body, $encoding = '')
```
Comment (Jamesking56) [inline tests/_support/Helper/Acceptance.php:9]:
```suggestion
    public function sendEmail($to, $subject, $body, $encoding = null)
```

**Comment by Jamesking56** [comment]
Thanks @margori, if you fix my final comments then I think its good to go in :+1:

**Comment by margori** [comment]
Done

**Pull request merged**